web/templates/skipped_marking_unread.hbs

```diff
@@ -0,0 +1,15 @@
+{{#if stream_names_with_privacy_symbol_html.[1]}}
+    {{#tr}}
+        Because you are not subscribed to <z-streams></z-streams>, messages in these streams were not marked as unread.
```

Comment:
I think would be better to have stream names like "stream1, stream2 and stream3".